File: LiferayScreensReactNative/ios/LiferayScreenlets/Auth/ForgotPassword/ForgotPasswordScreenletManager.m
//
//  ForgotPasswordScreenletManager.m
//  LiferayScreensReactNative
//
//  Created by Luis Miguel Barco on 13/08/2018.
//  Copyright © 2018 Facebook. All rights reserved.
//

#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>
#import <React/RCTEventEmitter.h>
#import <React/RCTView.h>

@interface RCT_EXTERN_MODULE(ForgotPasswordScreenletManager, RCTViewManager)

// Events
RCT_EXPORT_VIEW_PROPERTY(onForgotPasswordSent, RCTBubblingEventBlock)
RCT_EXPORT_VIEW_PROPERTY(onForgotPasswordError, RCTBubblingEventBlock)

// Properties
RCT_EXPORT_VIEW_PROPERTY(screenletAttributes, NSDictionary)

@end

File: LiferayScreensReactNative/ios/LiferayScreenlets/Auth/ForgotPassword/ForgotPasswordScreenletManager.swift
//
//  ForgotPasswordScreenletManager.swift
//  LiferayScreensReactNative
//
//  Created by Luis Miguel Barco on 13/08/2018.
//  Copyright © 2018 Facebook. All rights reserved.
//

@objc(ForgotPasswordScreenletManager)
class ForgotPasswordScreenletManager: RCTViewManager {
  override func view() -> UIView! {
    return ForgotPasswordScreenletView(frame: .zero)
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return false
  }
}


